src/app/(app)/missing-shoes/Components/MissingShoesFormBuilder.tsx

```diff
@@ -0,0 +1,261 @@
+"use client";
+
+import React from "react";
+import FormWrapper from "@/common/FormWrapper";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import { z } from "zod";
+import { Button } from "@/components/ui/button";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+import { FormTitle } from "@/common/FormTitle/FormTitle";
+import { Input } from "@/components/ui/input";
+import { useMask } from "@react-input/mask";
+import { DateTimePicker } from "@/common/DateTimePicker/DateTimePicker";
+import { CNIC_VALIDATOR_REGEX, PHONENUMBER_VALIDATOR_REGEX } from "@/lib/regex";
+import { ShoesTyes } from "@/constant/constant";
+
+export const missingShoesSchema = z.object({
+  color: z.string().min(1, {
+    message: "Color must be selected.",
```

Comment:
it is correct message , can we write that `color must be written `

Reply:
I have corrected this message